lib/search.coffee

```diff
@@ -126,22 +129,39 @@ createSearch = ({neighborhood})->
       return
     $('.incremental-search').remove()
     tally = searchResults.tally
-    resultPage = newPage()
-    resultPage.setTitle "Search for '#{searchQuery}'"
-    resultPage.addParagraph """
+    resultPage = {}
+    resultPage.title = "Search for '#{searchQuery}'"
+    resultPage.story = []
+    resultPage.story.push({
+      'type': 'paragraph'
+      'id': random.itemId()
+      'text': """
         String '#{searchQuery}' found on #{tally.finds||'none'} of #{tally.pages||'no'} pages from #{tally.sites||'no'} sites.
         Text matched on #{tally.title||'no'} titles, #{tally.text||'no'} paragraphs, and #{tally.slug||'no'} slugs.
         Elapsed time #{tally.msec} milliseconds.
-    """
+    """ })
     for result in searchResults.finds
-      resultPage.addItem
+      resultPage.story.push({
         "type": "reference"
         "site": result.site
         "slug": result.page.slug
         "title": result.page.title
         "text": result.page.synopsis || ''
+      })
 
-    link.showResult resultPage
+    resultPage.journal = [{
+      "type": "create",
+      "item": {
+        "title": resultPage.title,
+        "story": deepCopy(resultPage.story)
+      },
+      "date": Date.now()
+    },{
+      "type": "fork",
+      "date": Date.now()
+    }]
```

Comment:
I'm trying to remember the logic for adding this extra fork. Our delete logic is based on viewing the first revision recalled from history. A page with only the create in the journal is not subject to the delete logic.
https://github.com/fedwiki/wiki-client/blob/f3f7043ca09456e01b6331abdea9b9e916ed265d/lib/legacy.coffee#L242

I can remember having some trouble that was resolved by adding a fork from the ui such that some subsequent operation was made visible. I can't put my finger on it. But I don't think anticipating this situation and adding the extra action is necessary. Here are the normal cases that will work fine without it.

- ghost page with create only appears
- ghost page is abandoned with no impact on the server
- ghost page is forked to live on in the server, now has two actions in the journal
- page living on in the server is to be discarded, open create action and fork, done.

Reply:
> A page with only the create in the journal is not subject to the delete logic.

Only this is not what happens to a ghost page with only a create action.